imap_processing/tests/hit/test_hit_l1b.py

```diff
@@ -2,51 +2,174 @@
+        "ebox_5d1vd",
+        "active_heater",
+        "last_error_num",
+        "last_bad_cmd",
+        "ref_p5v",
+        "code_checksum",
+        "mode",
+    }
+    # Check that the dataset has the correct variables
+    assert valid_keys == set(hk_dataset.data_vars.keys())
+    assert set(dropped_keys).isdisjoint(set(hk_dataset.data_vars.keys()))
 
-    Creates a xarray dataset for housekeeping data
-    """
 
-    # create the attribute manager for this data level
-    attr_mgr = ImapCdfAttributes()
-    attr_mgr.add_instrument_global_attrs(instrument="hit")
-    attr_mgr.add_instrument_variable_attrs(instrument="hit", level="l1b")
-    attr_mgr.add_global_attribute("Data_version", "001")
+def test_hit_l1b_hk_dataset_attributes(hk_dataset):
+    """Test the attributes, dims, and coords in the housekeeping dataset"""
+    # Define the housekeeping dataset attributes
+    dataset_attrs = {
+        "Data_level": "1B",
+        "Data_type": "L1B_HK>Level-1B Housekeeping",
+        "Data_version": "001",
+        "Descriptor": "HIT>IMAP High-energy Ion Telescope",
+        "Discipline": "Solar Physics>Heliospheric Physics",
+        "File_naming_convention": "source_descriptor_datatype_yyyyMMdd_vNNN",
+        "HTTP_LINK": "https://imap.princeton.edu/",
+        "Instrument_type": "Particles (space)",
+        "LINK_TITLE": "IMAP The Interstellar Mapping and Acceleration Probe",
+        "Logical_file_id": None,
+        "Logical_source": "imap_hit_l1b_hk",
+        "Logical_source_description": "IMAP Mission HIT Instrument Level-1B "
+        "Housekeeping Data.",
+        "Mission_group": "IMAP",
+        "PI_affiliation": "Princeton University",
+        "PI_name": "Prof. David J. McComas",
+        "Project": "STP>Solar Terrestrial Probes",
+        "Source_name": "IMAP>Interstellar Mapping and Acceleration Probe",
+        "TEXT": "The High-energy Ion Telescope (HIT) measures the elemental "
+        "composition, energy spectra, angle distributions, and arrival "
+        "times of high-energy ions. HIT delivers full-sky coverage from "
+        "a wide instrument field-of-view (FOV) to enable a high resolution "
+        "of ion measurements, such as observing shock-accelerated ions, "
+        "determining the origin of the solar energetic particles (SEPs) "
+        "spectra, and resolving particle transport in the heliosphere. "
+        "See https://imap.princeton.edu/instruments/hit for more details.\n",
+    }
 
-    l1b_hk_dataset = hit_l1b.create_hk_dataset(attr_mgr)
-    assert isinstance(l1b_hk_dataset, xr.Dataset)
+    # Define the coordinates and dimensions. Both have equivalent values
+    dataset_coords_dims = {"epoch", "adc_channels", "adc_channels_label"}
 
+    # Check that the dataset has the correct attributes, coordinates, and dimensions
+    assert hk_dataset.attrs == dataset_attrs
+    assert hk_dataset.coords.keys() == dataset_coords_dims
+    assert hk_dataset.sizes.keys() == dataset_coords_dims
```

Comment:
I'm a little confused about how the dataset_coords_dims dictionary tests the `.sizes`. Doesn't sizes return some number for the dataset size?